src/commands/publish/link-publish-mode.ts

```diff
@@ -54,20 +62,68 @@ export const linkPublishMode = async (
 		return;
 	}
 
-	/**
-	 * If it's a symlink, make sure it's in the node_modules directory of the base package.
-	 * e.g. This could happen with pnpm
-	 *
-	 * If it's not, it might be a development directory and we don't want to overwrite it.
-	 */
-	const linkPathReal = await fs.realpath(linkPath);
-	if (!linkPathReal.startsWith(expectedPrefix)) {
-		return;
-	}
-
-	await hardlinkPackage(
+	const throttledHardlinkPackage = throttle(hardlinkPackage, 500);
```

Comment:
It's not immediately clear to me why you chose to throttle this function. Maybe a code comment would be a good idea here.

Reply:
I also was curious so I investigated a bit the difference between [throttleit](https://github.com/sindresorhus/throttleit?tab=readme-ov-file) and [p-throttle](https://github.com/sindresorhus/p-throttle). I _think_ `throttleit` is appropriate here 👍🏼, since I assume we want to discard any but the last invocation of `hardlinkPackage` in an interval